bot.py

```diff
@@ -96,12 +96,16 @@ def __init__(self, guild, loop, dispatcher_user):
             "resume", handler=self.resume, guarded_by=self.command_lock
         )
         self.register_command("skip", handler=self.skip, guarded_by=self.command_lock)
+        self.register_command("next", handler=self.skip, guarded_by=self.command_lock)
         self.register_command(
             "disconnect", handler=self.disconnect, guarded_by=self.command_lock
         )
         self.register_command(
-            "queue", handler=self.show_queue, guarded_by=self.command_lock
```

Comment:
Please make sure that removing the lock here doesn't reintroduce a bug. 
See my commit: f74937ce4d9fbb31066676dca35d81dd8cab0be4
And the third point of the PR message on #31

Reply:
Added back